IonDotnet/Timestamp.cs

```diff
@@ -361,7 +361,14 @@ private static int GetOffsetMinutes(string s, int startIdx)
                 throw new FormatException(s);
             }
 
-            return hour * 60 + minute;
+            int offset = hour * 60 + minute;
+
+            if (hour > 23 || minute > 59)
+            {
+                throw new FormatException($"{s} - Offset { offset } must be between -1439 and 1439 inclusive and minute between 00 and 59.");
```

Comment:
Is the message "minute between 00 and 59" necessary? It is implied with the Offset must be between -1439 and 1439

Reply:
The reason I include that in the message is that, technically "2007-01-01T00:00-10:70" is between -1439 and 1439 but not valid.

Reply:
In amazon interface they throw this message in such cases: "Minute offset 70 above maximum or equal to : 60"
Yet I am not a very strong opinion about it, I won't mind changing it if you think it's better.

Reply:
I don't have strong opinion about this either. I think we can leave it as is.